### PR TITLE
Chapter_1_3 finished

### PR DESCRIPTION
**Pull request by DaddyWesker**
@vsbogd

### CODE REVIEW

File: SICP_book/chapter_1_3.metta [hunk 1]
```diff
@@ -498,5 +480,332 @@
...
+
+(= (n_root $n $x)
+  (fixed-point-of-transform (lambda1 $y (/ $x (rpow $y (dec $n))))
+                            (repeated_f average-damp (closest_pow2 $n))
+                            1))
+
+; FIXME: Requires correct lambda2 type to work (see note to the lambda2 definition above). Works currently with no typedef.
+;!(assertEqual
+;    (n_root 5 8)
+;    1.5170557601762498)
```
Comment (vsbogd) [inline SICP_book/chapter_1_3.metta:766]: Looks like this one doesn't work neither with nor without type definition.

File: SICP_book/chapter_1_3.metta [hunk 2]
```diff
@@ -498,5 +480,332 @@
...
+;
+; --------------------------------------------------------
+
+(= (cubic $a $b $c)
+  (lambda1 $x (+4 (cube $x) (* $a (sqr $x)) (* $b $x) $c)))
+
+; FIXME: Requires correct lambda2 type to work (see note to the lambda2 definition above). Works currently with no typedef.
+;!(assertEqual
+;    (newtons-method (cubic 1 2 3) 1)
+;    -1.2762090909231458)
```
Comment (vsbogd) [inline SICP_book/chapter_1_3.metta:638]: Not sure it works without lambda type definitions. Did you check this?

**Comment by DaddyWesker** [comment]
@vsbogd  all those assertEquals works with or without lambda2 typedef. Though I haven't  tested them on a current main. I was using like week+ old metta. I'll check and post result here. 

I was thinking on splitting this chapter to two parts, one will contain only code working with no lambda2 typedef, other part will contain all the other examples. But this will break exercise order. What do you think?

**Comment by DaddyWesker** [comment]
Ok. I'll split chapter then.

**Comment by DaddyWesker** [comment]
Okay, so I've splitted chapter 1.3 to two parts. (newtons-method (cubic 1 2 3) 1) works for me. But n_root for some reason refused to work. I've spent several hours to understand why and still no clue. It worked previously. Maybe on the previous build of metta. 

Also, exercise 1.36 currently not working. It shows intermediate approximations but on the last step it fails. And here I've spent some time too. So, currently it is FIXME. I'll try to understand what is wrong.

**Pull request merged**